core/src/main/scala/org/broadinstitute/dsde/rawls/webservice/SnapshotApiService.scala

```diff
@@ -69,7 +70,6 @@ trait SnapshotApiService extends UserInfoDirectives {
         parameters("offset".as[Int], "limit".as[Int]) { (offset, limit) =>
           complete {
             snapshotServiceConstructor(userInfo).enumerateSnapshots(WorkspaceName(workspaceNamespace, workspaceName), offset, limit)
-              .map(resourceListToDataReferenceList)
```

Comment:
We should delete resourceListToDataReferenceList in the unlikely event we replace v2 enumerate snapshots in place.